runners/wikiEclipse/templates/createDevon4jProject.asciidoc

```diff
@@ -0,0 +1,42 @@
+== Create a new Devon4j Project
+
+=== Create a new Devon4j Project in the Eclipse IDE
+
+==== Prerequisites
+
+In order to create a new application you must use the archetype provided by devon4j which uses the https://maven.apache.org/guides/introduction/introduction-to-archetypes.html[maven archetype functionality].
+
+To create a new application, you should have installed devonfw IDE. Follow the https://devonfw.com/website/pages/docs/devonfw-ide-introduction.asciidoc.html[devonfw-ide] documentation to install the same.
+
+==== Create the Project
+
+Firstly, create a new Maven Project.
+Then, choose the 'devon4j-template-server' archetype.
+Fill the Group Id, Artifact Id, Version and Package for your project. The package name should be <%= name; %>.
```

Comment:
Maybe try to highlight the labels of some text field like  *Group Id* for better visibility where the user has to insert something.